client/admin-pointers.js

```diff
@@ -0,0 +1,26 @@
+/*global wcSevicesAdminPointers, ajaxurl */
+import jQuery from 'jquery';
+
+jQuery( document ).ready( function( $ ) {
+	const pointers = wcSevicesAdminPointers;
+
+	function show_pointer( i ) {
```

Comment:
The index is required in order to show the next available pointer. I didn't want to use an external counter since the call to `show_pointer()` to show the next pointer seemed intuitive enough. However, if that's not clear, I can make the change.